### PR TITLE
attach_detach_disk: Update judgement for shareable test

### PR DESCRIPTION
**Pull request by hahakiki2010**
Shareable options is introduced afterwards libvirt 3.9.0.
So if libvirt version below 3.9.0, should cancel related test cases
otherwise it would get failure like
"vm start failed. lockIs another process using the image"

Signed-off-by: Yan Li <yannli@redhat.com>

### CODE REVIEW

**Comment by hahakiki2010** [comment]
(.libvirt-ci-venv-ci-runtest-CKXuxQ) [root@dell-per720-03 domain]# avocado run --vt-type libvirt virsh.attach_detach_disk.attach_disk.normal_test.twice_diff_target
JOB ID     : c062e83c5d42ca7a153e5c21bb62a0ff633d0ea1
JOB LOG    : /root/avocado/job-results/job-2018-06-12T07.25-c062e83/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.attach_detach_disk.attach_disk.normal_test.twice_diff_target: PASS (42.28 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 44.44 s
(.libvirt-ci-venv-ci-runtest-CKXuxQ) [root@dell-per720-03 domain]# avocado run --vt-type libvirt virsh.attach_detach_disk..twice_multifunction_with_shareable
JOB ID     : a44da56265787e86ed1cfc934687b52958b3f102
JOB LOG    : /root/avocado/job-results/job-2018-06-12T07.26-a44da56/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.attach_detach_disk.attach_disk.normal_test.twice_multifunction_with_shareable: PASS (59.25 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 61.42 s

File: libvirt/tests/src/virsh_cmd/domain/virsh_attach_detach_disk.py [hunk 1]
```diff
@@ -103,6 +103,20 @@ def acpiphp_module_modprobe(vm, os_type):
...
+
+        at_with_shareable: True or False. Whether attach disk with shareable option
+        test_twice: True or False. Whether perform operations twice
+        return: True or cancel the test
+        """
+        if at_with_shareable or test_twice:
+            if libvirt_version.version_compare(3, 9, 0):
+                return True
+            else:
+                test.cancel("Current libvirt version doesn't support shareable feature")
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_attach_detach_disk.py]: I assume before 3.9.0, shareable attribute can be also supported,please confirm with yanjing

**Comment by iccaszhulili** [comment]
rh-caselink retest please

File: libvirt/tests/src/virsh_cmd/domain/virsh_attach_detach_disk.py (hunk 1 above)
Comment (kylazhang) [reply libvirt/tests/src/virsh_cmd/domain/virsh_attach_detach_disk.py]: missing default return value

**Pull request merged**